common/spring-cloud-stream-modules-common-configuration/src/test/java/org/springframework/cloud/stream/module/ftp/FtpSessionFactoryPropertiesTests.java

```diff
@@ -98,6 +99,7 @@ public void noPropertiesThrowsMeaningfulException() {
 			context.register(Conf.class);
 			context.refresh();
 			fail("should throw exception");
+			context.close();
```

Comment:
It's just there to stop STS (eclipse) from barking at me about a `Closeable` that's not closed. I just hate yellow squiggly lines :smile: 

It could go before the `fail`, I guess but then it might imply it's close the that's expected to fail.